experimenter/experimenter/experiments/tests/test_admin.py

```diff
@@ -208,6 +208,7 @@ def test_resource_dehydrate_none(self):
         experiment.reference_branch = None
         experiment.status_next = ""
         experiment.conclusion_recommendation = ""
+        experiment.qa_status = None
```

Comment:
I think from your previous changes, it should be set to `Not set`

Reply:
Oh yeah that's a good point, that field should be not null now and fall back to the 'not set' state

Reply:
That will all be updated when the other one lands 👍 Whichever one lands first, this will be fine